src/cli/tutorial.rs

```diff
@@ -0,0 +1,66 @@
+use crate::state::State;
+use async_trait::async_trait;
+use nu_cli::OutputStream;
+use nu_engine::CommandArgs;
+use nu_errors::ShellError;
+use nu_protocol::{Signature, SyntaxShape};
+use std::sync::Arc;
+
+pub struct Tutorial {
+    state: Arc<State>,
+}
+
+impl Tutorial {
+    pub fn new(state: Arc<State>) -> Self {
+        Self { state }
+    }
+}
+
+#[async_trait]
+impl nu_engine::WholeStreamCommand for Tutorial {
+    fn name(&self) -> &str {
+        "tutorial"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("tutorial").optional(
+            "action",
+            SyntaxShape::String,
+            "the action to perform - one of next or prev",
+        )
+    }
+
+    fn usage(&self) -> &str {
+        "Perform document operations against a bucket or collection"
```

Comment:
is that usage copied from somwhere? Maybe that it is a tutorial might be a better description? (since it might also introduce more concepts around nushell etc)